gems/pending/postgres_ha_admin/failover_databases.rb

```diff
@@ -0,0 +1,78 @@
+require 'util/postgres_dsn_parser'
+require 'postgres_ha_admin/postgres_ha_logger'
+require 'pg'
+
+module PostgresHaAdmin
+  class FailoverDatabases
+    include PostgresHaAdmin
+
+    attr_reader :yml_file, :connection_hash, :log, :connection_hash
+
+    def initialize(config_dir, log_dir, connection_params_hash)
+      @yml_file = Pathname.new(config_dir).join(DB_YML_FILE)
```

Comment:
@jrafanie I was planning to put more code in ```postgres_ha_admin.rb```` for next task in HA and refactor/fix, if needed, in the next PR